README.md

```diff
@@ -31,47 +29,39 @@ $ git submodule update
 ```
 ## Compilation:
 **1** Install 3d-parties developers libraries
+- Run the following commands :
 ```
-sudo apt-get install liblua5.2-dev libxml2-dev lua-lpeg-dev
-sudo apt-get install openssl
-```
-
-**2** Install Qt5.3+
-
-[Download Qt5.3+](https://download.qt.io/archive/qt/5.3/5.3.1/)
-```
-sudo chmod +x some-qt-build.run
-./some-qt-build.run
+$ sudo apt-get install liblua5.2-dev libxml2-dev lua-lpeg-dev
+$ sudo apt-get install openssl
 ```
 
-**2** Setup QMAKE environment variable to path to qmake
-```export QMAKE=${PATH_TO_QMAKE} ```
-*You can get path to qmake this way:*
+**2** Install Qt5.9+
+- Run the following commands :
 ```
-$ sudo find / -name qmake
-/usr/bin/qmake
-/opt/Qt5.3.1/5.3/gcc/bin/qmake
+$ sudo add-apt-repository -y ppa:beineri/opt-qt591-xenial
```

Comment:
This command fails on my end
```
Err:14 http://ppa.launchpad.net/beineri/opt-qt591-xenial/ubuntu bionic Release      
 404  Not Found [IP: 91.189.95.83 80]
...
E: The repository 'http://ppa.launchpad.net/beineri/opt-qt591-xenial/ubuntu bionic Release' does not have a Release file
``` so I am not able to use this ppa to install the `qt59` packages.

Could you please confirm that it works on your end? If not then the install steps should be changed

Reply:
@ShobhitAd QT5.9 is not required for this proposal, so the changes were reverted back in ff031b6